attractions/slider/types.ts

```diff
@@ -0,0 +1,21 @@
+export type TickConfig =
+  | { mode: 'none' }
+  | {
+      mode: 'steps';
+      step: number;
+      subDensity?: number;
+    }
+  | {
+      mode: 'values';
+      values: number[];
+      subDensity?: number;
+    };
+export type RangeBehavior = 'free' | 'block' | 'push';
+export type SliderState = {
+  value: number;
+  activeHandle: number;
+  min: number;
+  max: number;
+  sliderActive: boolean;
+};
+export type Tooltips = 'active' | 'always' | 'never' | undefined;
```

Comment:
this is one possibility, to not pass a value for Tooltips, ie undefined. It is a bit confusing to me since we are not using Typescript...if this was a props interface you would use ? to represent this.